### PR TITLE
test: extract wrappers for map integration tests

### PR DESCRIPTION
**Pull request by sissbruecker**
## Description

Extract wrappers for accessing OpenLayers configuration instances from integration tests, and adds a few more test cases.

TODO:
- [x] Add JavaDoc for wrappers

### CODE REVIEW

File: vaadin-map-flow-parent/vaadin-map-flow-integration-tests/src/main/java/com/vaadin/flow/component/map/MarkerFeaturePage.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package com.vaadin.flow.component.map;
+
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.NativeButton;
+import com.vaadin.flow.component.map.configuration.Coordinate;
+import com.vaadin.flow.component.map.configuration.Feature;
```
[the remark this answers is not in the document]
Comment (sissbruecker) [reply vaadin-map-flow-parent/vaadin-map-flow-integration-tests/src/main/java/com/vaadin/flow/component/map/MarkerFeaturePage.java]: Removed

File: vaadin-map-flow-parent/vaadin-map-flow-integration-tests/src/main/java/com/vaadin/flow/component/map/MarkerFeaturePage.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import com.vaadin.flow.component.map.configuration.feature.MarkerFeature;
+import com.vaadin.flow.component.map.configuration.style.Icon;
+import com.vaadin.flow.router.Route;
+
+@Route("vaadin-map/marker-feature")
+public class MarkerFeaturePage extends Div {
+    public MarkerFeaturePage() {
+        Map map = new Map();
+        map.setWidthFull();
+        map.setHeight("400px");
```
[the remark this answers is not in the document]
Comment (sissbruecker) [reply vaadin-map-flow-parent/vaadin-map-flow-integration-tests/src/main/java/com/vaadin/flow/component/map/MarkerFeaturePage.java]: Removed from all test pages / demos that don't need a specific size

File: vaadin-map-flow-parent/vaadin-map-testbench/src/main/java/com/vaadin/flow/component/map/testbench/MapElement.java [hunk 3]
```diff
@@ -114,4 +72,385 @@ public void disableInteractions() {
...
+     * the object (see {@link #get(String, Object...)} and related methods). All
+     * properties are evaluated lazily using an {@link ExpressionExecutor},
+     * which takes the path through the configuration hierarchy to that property
+     * as Javascript expression and executes it through the Selenium API.
+     */
+    public static abstract class ConfigurationObjectReference {
+        ExpressionExecutor executor;
+        String expression;
+
+        public ConfigurationObjectReference(ExpressionExecutor executor,
```
[the remark this answers is not in the document]
Comment (sissbruecker) [reply vaadin-map-flow-parent/vaadin-map-testbench/src/main/java/com/vaadin/flow/component/map/testbench/MapElement.java]: Good point. I had some vague idea about allowing developers to extend the wrappers, but that wasn't really thought through. For now I just made all constructors private.

**Review by DiegoCardoso — APPROVED** [review]
LGTM 🎉

**Pull request merged**